src/frontend/components/registrationForm.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from "react";
+import Input from "./input";
+import Button from "./button";
+import Select from "./Select";
+
+class RegistrationForm extends Component {
+  constructor(props) {
+    super(props);
+  }
+  handleChange = () => {
+    console.log("Changes made in input ");
+  };
+  handleSubmit = () => {
+    console.log("Submit button clicked");
+  };
+  getDays = () => {
+    const days = [];
+    for (let i = 1; i <= 31; i++) {
+      days.push(i);
+    }
+    return days;
+  };
+  getMonths = () => {
+    var months = [
+      "Jan",
+      "Feb",
+      "Mar",
+      "Apr",
+      "May",
+      "Jun",
+      "Jul",
+      "Aug",
+      "Sep",
+      "Oct",
+      "Nov",
+      "Dec"
+    ];
+    return months;
+  };
+  getYear = () => {
+    const years = [];
+    const currentYear = new Date().getFullYear();
+    for (let i = 1920; i <= currentYear; i++) {
+      years.push(i);
+    }
+    return years;
+  };
+  render() {
+    return (
+      <div className="form-component">
+        <div className="form-text">
+          <p>Create a Free Account</p>
+        </div>
+        <Input
+          type="text"
+          name="fullname"
+          value="fullname"
+          placeholder="First name and Last name(s)"
+          onChange={this.handleChange}
+        />
+        <Select
```

Comment:
It would be helpful if we had the options element. That would allow us to add a default option e.g. `Select Day`. Also, don't forget to add the `keys` prop when you generate the options element from the array.